instrumentation/instagin/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/instana/go-sensor/instrumentation/instagin
+
+go 1.11
+
+require (
+	github.com/gin-gonic/gin v1.7.2
+	github.com/instana/go-sensor v1.29.1-0.20210608074124-c12a7d59b2d4
```

Comment:
Let's not forget to update it to a released version before merging.

Reply:
👍